finetune/util/huggingface_interface.py

```diff
@@ -16,6 +19,19 @@
 
 from tensorflow.python.util import tf_inspect
 
+def strip_accents(text):
+    """
+    https://stackoverflow.com/questions/517923/what-is-the-best-way-to-remove-accents-normalize-in-a-python-unicode-string
+    """
+    try:
+        text = unicode(text, 'utf-8')
+    except (TypeError, NameError): # unicode is a default on python 3 
+        pass
+    text = unicodedata.normalize('NFD', text)
+    text = text.encode('ascii', 'ignore')
```

Comment:
We strip accents by encoding as ascii?  This seems concerning

Reply:
This is only used for aligning the output of the tokenizer with the text. We need some processing that is at least as destructive as any processing the tokenizer does. and where ```norm(text) == norm(tokenizer_norm(text))```  With this in mind I don't think there is anything wrong with this for tokenisers where this holds.
I would propose that we rename this to something more indicative of it's intention and make it opt-in per tokenizer that we wrap. What do you think?

Reply:
How were we previously dealing with accents? Leaving them in the vocabulary?

Reply:
This is processing only applies to the output of Huggingface models that use a sentencepiece based tokenizer. To allow us to align the tokens and get start and end char indexes. - Primarily required for Albert.
Albert's tokenizer removes accents in what is returned for the token text, meaning our previous approach to align it and get start and end indexes was failing silently.

Reply:
Got it -- didn't realize that this was only for alignment.  I think with that in mind, I have no problems with this.